ibis/backends/postgres/tests/test_client.py

```diff
@@ -137,7 +137,7 @@ def test_create_and_drop_table(con, temp_table, params):
 
     con.drop_table(temp_table, **params)
 
-    with pytest.raises(com.IbisError):
+    with pytest.raises(com.TableNotFound, match="Table not found"):
```

Comment:
```suggestion
    with pytest.raises(com.TableNotFound, match=temp_table):
```

Probably want to check that the table name is mentioned, not only that the message says "Table not found".